src/client/Microsoft.Identity.Client/ManagedIdentity/AzureArcManagedIdentitySource.cs

```diff
@@ -81,13 +81,16 @@ private AzureArcManagedIdentitySource(Uri endpoint, RequestContext requestContex
             }
         }
 
-        protected override ManagedIdentityRequest CreateRequest(string resource)
+        protected override ManagedIdentityRequest CreateRequest(string resource, AcquireTokenForManagedIdentityParameters parameters)
         {
             ManagedIdentityRequest request = new ManagedIdentityRequest(System.Net.Http.HttpMethod.Get, _endpoint);
 
             request.Headers.Add("Metadata", "true");
             request.QueryParameters["api-version"] = ArcApiVersion;
             request.QueryParameters["resource"] = resource;
+            request.QueryParameters["bypass_cache"] = "false";
+
+            ApplyClaimsAndCapabilities(request, parameters);
```

Comment:
Have Azure Arc sing off on a spec?

Reply:
Azure ARC will be on MSI v2 Token Revocation. This will be updated in the next commit